Source/WebCore/platform/graphics/controls/ControlStyle.h

```diff
@@ -56,7 +57,9 @@ struct ControlStyle {
     OptionSet<State> states;
     unsigned fontSize { 12 };
     float zoomFactor { 1 };
+    Color actionColor;
```

Comment:
Is this specific to `MenuListButton`?

Reply:
No it is also used by `SliderTrackPart::drawTicks(`). It is now called `textColor`.